provider/cmd/pulumi-resource-aws/schema.json

```diff
@@ -245414,7 +245414,7 @@
                 },
                 "forceNewDeployment": {
                     "type": "boolean",
-                    "description": "Enable to force a new task deployment of the service. This can be used to update tasks to use a newer Docker image with same image/tag combination (e.g., `myimage:latest`), roll Fargate tasks onto a newer platform version, or immediately deploy `ordered_placement_strategy` and `placement_constraints` updates.\n"
+                    "description": "Enable to force a new task deployment of the service. This can be used to update tasks to use a newer Docker image with same image/tag combination (e.g., `myimage:latest`), roll Fargate tasks onto a newer platform version, or immediately deploy `ordered_placement_strategy` and `placement_constraints` updates.\nWhen using the forceNewDeployment property you also need to configure the Triggers property.\n"
```

Comment:
Nit: we have two different styles of property naming in the description now. (`placement_constraints` vs `forceNewDeployment`)

Reply:
In my understanding forceNewDeployment is the preferred style when we cannot inflect toward the target language.